docs/reference/manifest/appdomain.md

```diff
@@ -7,14 +7,16 @@ Specifies an additional domain that will be used to load pages in the add-in win
 ## Syntax
 
 ```XML
-<AppDomain>string </AppDomain>
+<AppDomain>string</AppDomain>
 ```
 
+> [!IMPORTANT]
+> The **AppDomain** must include the protocol e.g., `<AppDomain>https://myappdomain"<AppDomain>`.
```

Comment:
Suggest rephrasing to -> The value of the **AppDomain** element must include the protocol (e.g., `<AppDomain>https://myappdomain<AppDomain>`).

- puts "e.g.,..." in parenthesis
- removes the erroneous quotation mark from the end of the specified domain